src/bookmarked.html

```diff
@@ -17,7 +17,80 @@
     <header class="app-header">
       <h1 class="app-header__heading">Bookmarked</h1>
     </header>
-    <main class="app-main"></main>
+    <main class="app-main">
+      <section class="quiz-card">
+        <h2 class="quiz-card__heading">Question</h2>
+        <button class="quiz-cart__button">Show answer</button>
+        <p class="quiz-card__answer">
+          This is the answer of the question above!
+        </p>
+        <ul class="quiz-card__taglist">
+          <li>#tag 1</li>
+          <li>#tag 2</li>
+          <li>#tag 3</li>
+          <li>#tag 4</li>
+        </ul>
+        <div
+          role="button"
+          class="quiz-card__bookmark quiz-card__bookmark--added"
+          aria-label="Add bookmark"
+        ></div>
+      </section>
+      <section class="quiz-card">
+        <h2 class="quiz-card__heading">Question</h2>
+        <button class="quiz-cart__button">Show answer</button>
+        <p class="quiz-card__answer">
+          This is the answer of the question above!
+        </p>
+        <ul class="quiz-card__taglist">
+          <li>#tag 1</li>
+          <li>#tag 2</li>
+          <li>#tag 3</li>
+          <li>#tag 4</li>
+        </ul>
+        <div
+          role="button"
+          class="quiz-card__bookmark quiz-card__bookmark--added"
+          aria-label="Add bookmark"
+        ></div>
+      </section>
+      <section class="quiz-card">
+        <h2 class="quiz-card__heading">Question</h2>
+        <button class="quiz-cart__button">Show answer</button>
+        <p class="quiz-card__answer">
+          This is the answer of the question above!
+        </p>
+        <ul class="quiz-card__taglist">
+          <li>#tag 1</li>
+          <li>#tag 2</li>
+          <li>#tag 3</li>
+          <li>#tag 4</li>
+        </ul>
+        <div
+          role="button"
+          class="quiz-card__bookmark quiz-card__bookmark--added"
+          aria-label="Add bookmark"
+        ></div>
+      </section>
+      <section class="quiz-card">
+        <h2 class="quiz-card__heading">Question</h2>
+        <button class="quiz-cart__button">Show answer</button>
+        <p class="quiz-card__answer">
+          This is the answer of the question above!
+        </p>
+        <ul class="quiz-card__taglist">
+          <li>#tag 1</li>
+          <li>#tag 2</li>
+          <li>#tag 3</li>
+          <li>#tag 4</li>
+        </ul>
+        <div
+          role="button"
+          class="quiz-card__bookmark quiz-card__bookmark--added"
+          aria-label="Add bookmark"
+        ></div>
+      </section>
+    </main>
     <footer class="navbar-main">
       <nav>
         <button class="navbar__button" aria-label="home">Home</button>
```

Comment:
The navigation should contain anchor elements which will link to the designated HTML file.
```suggestion
        <a href="/" class="navbar__button" aria-label="home">Home</button>
```